api/ScreenDetailed.json

```diff
@@ -34,7 +34,7 @@
       },
       "availLeft": {
         "__compat": {
-          "spec_url": "https://w3c.github.io/window-management/#ref-for-dom-screendetailed-availleft",
```

Comment:
Why is this change necessary? 

@sideshowbarker has opinions about where to link in the specs. I think it has to do with where the MDN ribbon appears.

Reply:
The practice followed by the majority of the existing spec URLs is to use the URL for the developer part of the spec, if any. In this case, that’s https://w3c.github.io/window-management/#ref-for-dom-screendetailed-availleft

And yeah the effect of that as far as the rendering of the published specs goes is that it causes the **MDN** annotation in the spec to be shown next the developer part of the spec (rather than the implementor part).